menas/ui/service/EntityService.js

```diff
@@ -325,7 +325,22 @@ class DatasetService extends EntityService {
     }
   }
 
+  // if _properties with empty string exist, they will be replaced with undefs in place (backend expecting an empty option)
```

Comment:
```suggestion
  // if properties with empty string exist, they will be replaced with undefs in place (backend expecting an empty option)
```

Reply:
No. Both `properties` and `_properties` are present in the object. The former holds the initial information, while the latter hold the submitted data. So `_properties` is, indeed, the desired filed to be adjusted.

Reply:
Missed that, sorry.